docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/LightGbmAdvanced.cs

```diff
@@ -33,26 +36,32 @@ public static void Example()
                 .Select(column => column.Name) // Get the column names
                 .Where(name => name != labelName) // Drop the Label
                 .ToArray();
-            var pipeline = mlContext.Transforms.Concatenate("Features", featureNames)
-                           .Append(mlContext.Regression.Trainers.LightGbm(
-                                            labelColumnName: labelName,
-                                            numberOfLeaves: 4,
-                                            minimumExampleCountPerLeaf: 6,
-                                            learningRate: 0.001));
+            var pipeline = mlContext.Transforms.Concatenate("Features",
+                               featureNames)
+                               .Append(mlContext.Regression.Trainers.LightGbm(
+                               labelColumnName: labelName,
+                               numberOfLeaves: 4,
+                               minimumExampleCountPerLeaf: 6,
+                               learningRate: 0.001));
```

Comment:
there are a lot of spaces in front of this

Reply:
```
            var pipeline = mlContext.Transforms.Concatenate("Features",
                   featureNames)
                   .Append(mlContext.Regression.Trainers.LightGbm(
                       labelColumnName: labelName,
                       numberOfLeaves: 4,
                       minimumExampleCountPerLeaf: 6,
                       learningRate: 0.001));
```